lib/modules/manager/vendir/__fixtures__/empty-directory.yaml

```diff
@@ -0,0 +1,3 @@
+apiVersion: vendir.k14s.io/v1alpha1
```

Comment:
please inline small fixtures

Reply:
Done

Reply:
I tried a bunch of different ways to get this inlined properly. However, I could not make it work. Every attempt at properly inlining it so json was passed resulted in type errors. If you have an example of how to inline it, or the code to do so, I'm happy to accept help.